<?xml version="1.0"?>
<launch>
    <arg name="map_frame" default="map"/>

    <node pkg="geographic_conversion" type="geographic_conversion_server_node" name="geographic_conversion_server_node" output="screen" respawn="true" respawn_delay="0">
        <param name="map_frame" value="$(arg map_frame)"/>
    </node>
</launch>